TESTS.md

```diff
@@ -86,9 +86,12 @@ You can also have a run configuration:
 * Click on "Junit Tests"
 * Give a name to the configuration, e.g. `JUnit tests`
 * As "Test Kind", choose "All in directory"
-* As folder, choose the following folder: `mapbox-gl-native/android/java/MapboxGLAndroidSDKTestApp/src/test/java`
+* As folder, choose the following folder: `${PROJECT_DIR}/mapbox-gl-native/android/java/MapboxGLAndroidSDKTestApp/src/androidTest/java`
```

Comment:
The folder should be `mapbox-gl-native-android/MapboxGLAndroidSDKTestApp/src/androidTest/java`